File: jewellery_erpnext/gurukrupa_exports/web_form/demo_for_web_form/demo_for_web_form.js
frappe.ready(function() {
	frappe.web_form.on('employee_code', (field, value) => {
		frappe.call({
			method: 'jewellery_erpnext.gurukrupa_exports.web_form.demo_for_web_form.demo_for_web_form.get_context',
			args: {
				'employee_code':value
			},
			callback: (data) => {
				if (data.message) {
					console.log(data.message)
					frappe.web_form.set_value(['employee_name'], [data.message[0]['employee_name']])
					frappe.web_form.set_value(['department'], [data.message[0]['department']])
					frappe.web_form.set_value(['designation'], [data.message[0]['designation']])
					frappe.web_form.set_value(['reports_to'], [data.message[0]['reports_to']])
				}
			},
		});
	});
})

File: jewellery_erpnext/gurukrupa_exports/web_form/demo_for_web_form/demo_for_web_form.py
import frappe

@frappe.whitelist()
def get_context(employee_code):
	employee_data = []
	for i in  frappe.get_list('Employee',['name','employee_name','department','designation','reports_to']):
		if i['name'] == employee_code:
			employee_data.append(i)
			break
	return employee_data
